start-service.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -xe
+source /home/ec2-user/.bash_profile
+cd /home/ec2-user/app/release
+nohup sudo bash create-swap.sh  > ../logs/run_swap.out 2> ../logs/run_swap.err &
+sudo yum install -y python3 python3-pip
+venv/bin/python3 setup/stanza_setup.py
```

Comment:
whats the benefit of running this separately from the app? will it try to download again once the app is up and running but will skip since its done?

Reply:
On starting the app, we are backgrounding this process. So if on startup, if there is an issue downloading stanza (for whatever reason) the build will still be successful.

From my understanding, by separating it out, if stanza download fails, we will get a build fail by having this separated. Then on app start, it still checks if the file has been downloaded, it has been, then continues on its way

Reply:
gotchya. i'm always torn on whats the best way to validate that your service is up and healthy after a deployment. should that fail the deployment if it isn't? there are other things that go into the app starting up besides stanza download that could fail in theory. we should noodle on this. `banter-core` can flop if the database migration fails or any other number of things.